### PR TITLE
Setup mobile app

### PR DESCRIPTION
**Pull request by jonathan-marmelab**
Setup mobile app environment

### CODE REVIEW

File: proxy/development/nginx.conf [hunk 1]
```diff
@@ -0,0 +1,94 @@
+user                       nginx;
```
Comment (djhi) [inline proxy/development/nginx.conf:1]: This file and the other have tabs that make them less readable imo. Nitpicking though so feel free to ignore

File: README.md [hunk 2]
```diff
@@ -32,28 +32,42 @@ make setup-env
 webapp/.env
 ```
 
-Once the env file has been set,you can either run the app in development mode using:
+Once the env file has been set,you can either run the app in development or poduction mode.
```
Comment (djhi) [inline README.md:35]:
```suggestion
Once the env file has been set,you can either run the app in development or production mode.
```

File: README.md [hunk 3]
```diff
@@ -32,28 +32,42 @@ make setup-env
...
+
+/!\ Note: The server automatically detects mobile devices and serves the mobile app automatically. You can test mobile
+mode using the developper tools from your browser.
```
Comment (djhi) [inline README.md:38]:
```suggestion
/!\ Note: The server automatically detects mobile devices and serves the mobile app automatically. You can test mobile mode using the developper tools from your browser.
```

**Review by djhi — CHANGES_REQUESTED** [review]

File: proxy/development/nginx.conf (hunk 1 above)
Comment (jonathan-marmelab) [reply proxy/development/nginx.conf:1]: Actually, I find it more readable when aligned with tab

**Review by djhi — APPROVED** [review]

File: proxy/development/nginx.conf (hunk 1 above)
Comment (djhi) [reply proxy/development/nginx.conf:1]: You're obviously wrong ^^

**Pull request merged**